org.gnome.Boxes.json

```diff
@@ -0,0 +1,323 @@
+        {
+            "name" : "libcacard",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://www.spice-space.org/download/libcacard/libcacard-2.5.3.tar.xz",
+                    "sha256": "243ff03c563a95faed497db7f524fcb34ccd6f388d1175ecf31c371a3188963b"
+                }
+            ]
+        },
+        {
+            "name" : "qemu",
+            "config-opts" : [
+                "--enable-kvm",
+                "--enable-spice",
+                "--enable-usb-redir",
+                "--enable-smartcard"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://download.qemu.org/qemu-2.11.1.tar.xz",
+                    "sha256" : "8a5145d1f8bd2eadc6776f3e13c68cd28d01349e30639bdbcb26ac588d668686"
+                }
+            ]
+        },
+        {
+            "name" : "freerdp",
+            "buildsystem" : "cmake",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "http://pub.freerdp.com/releases/freerdp-2.0.0-rc2.tar.gz",
+                    "sha256" : "bf080f71655a4996942436d5aacd5ce4afe57e546d4282fc0dc42fb80bd9da0c"
+                }
+            ]
+        },
+        {
+            "name" : "osinfo-db-tools",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://releases.pagure.org/libosinfo/osinfo-db-tools-1.1.0.tar.gz",
+                    "sha256": "a141cd2fc07c30d84801b5dbf6b11f2c2e708b0e81216277d052ac0b57fe546b"
+                }
+            ]
+        },
+        {
+            "name" : "libosinfo",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://releases.pagure.org/libosinfo/libosinfo-1.1.0.tar.gz",
+                    "sha256": "600f43a4a8dae5086a01a3d44bcac2092b5fa1695121289806d544fb287d3136"
+                },
+                {
+                    "type" : "file",
+                    "url" : "http://pciids.sourceforge.net/v2.2/pci.ids",
+                    "sha256" : "0d45bf3ededf5ea5e233dee368f13ed1bd3f4dce1958fd33ec0efea8ec99be28"
+                },
+                {
+                    "type" : "file",
+                    "url" : "http://www.linux-usb.org/usb.ids",
+                    "sha256" : "970b720d56beec35b7ef8bda02e8b85ab853b436f3cae4fed13b808e1a79fdd8"
+                }
+            ]
+        },
+        {
+            "name" : "osinfo-db",
+            "buildsystem" : "simple",
+            "builddir": true,
+            "build-commands" : [
+                "make",
+                "osinfo-db-import --dir /app/share/osinfo/ osinfo-db-*.tar.xz"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.com/libosinfo/osinfo-db"
```

Comment:
It should be pointing to specific commit.

Reply:
Actually for this one we would like to have it as it is because osinfo-db is just a database of operating systems, and having the latest entries on the build  would be an advantage of using Flatpak vs. distro packaging. Can it be?

Reply:
No. Everything needs to be reproducible which means specific commits. Nothing stopping you doing a lot of releases ;-) (there are some similar issues with things like lens db's etc. in other apps)

Reply:
ah, ok. :/